spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/AetherFactories.java

```diff
@@ -52,22 +47,12 @@ public static RepositorySystemSession newSession(RepositorySystem system, boolea
 		if (!workOffline) {
 			session.setUpdatePolicy(RepositoryPolicy.UPDATE_POLICY_ALWAYS);
 		}
+		session.setChecksumPolicy(RepositoryPolicy.CHECKSUM_POLICY_FAIL);
```

Comment:
I thought this would be more useful, as I always prefer it to fail as soon as it needs to.